config/config_test.go

```diff
@@ -34,6 +35,91 @@ func TestT(t *testing.T) {
 	TestingT(t)
 }
 
+func (s *testConfigSuite) TestNullableBoolUnmashal(c *C) {
+	var nb = nullableBool{false, false}
+	data, err := json.Marshal(nb)
+	c.Assert(err, IsNil)
+	err = json.Unmarshal(data, &nb)
```

Comment:
Do not test the data marshaling from the struct `nullableBool`, it will never happen in real world.
Here's a real-world test.
```go
data := []byte("true")
json.Unmarshal(data, &nb)
```